token/core/zkatdlog/nogh/service.go

```diff
@@ -21,10 +21,11 @@ import (
 )
 
 type TokenCommitmentLoader interface {
-	GetTokenCommitments(ids []*token3.ID) ([]*token.Token, error)
+	GetTokenOutputs(ids []*token3.ID) ([]*token.Token, error)
 }
 
 type QueryEngine interface {
+	IsPending(id *token3.ID) (bool, error)
```

Comment:
Maybe add a comment explaining that IsPending refers to the transaction creating the token?